test/functional/features/search.feature

```diff
@@ -104,7 +104,7 @@ Feature: Search on index
             | name          | category  | field     |
             | sum_age       | sum       | age       |
             | genders       | terms     | gender    |
-        When I execute it on the index named "my_index" for type "my_type"
+        When I execute it on the index named "my_in1dex" for type "my_type"
```

Comment:
Typo ?

Reply:
Sure. What concerns me is how did this happen to change, and even more concerning, how the fuck did it manage to pass tests ? Anayway, fixed

Reply:
Yup, I’ll definitely look into it.